spec/support/monitor/monitor_idv_steps.rb

```diff
@@ -41,9 +41,6 @@ def verify_identity_with_doc_auth
     click_on 'Continue'
     expect(page).to have_current_path('/verify/doc_auth/verify')
 
-    click_on 'Continue'
-    expect(page).to have_current_path('/verify/doc_auth/doc_success')
```

Comment:
Just ran into a smoke test flake and I'm very glad we're updating this here :]

Reply:
> Just ran into a smoke test flake and I'm very glad we're updating this here :]

I don't know that I'd have a ton of confidence that this won't just flake on the next screen, but at least one fewer assertion that can go awry. 😄